index.js

```diff
@@ -6,15 +6,185 @@ var phantomjs = require('phantomjs');
+                return index % 2 === 0;
+            });
+            res.json({ error: 0, message: "data found", data: result });
+        } else {
+            res.json({ error: 1, message: "data not found", data: result });
         }
     })
-});
+})
+
+router.get('/get_eth/:start_date/:end_date', function(req, res, next) {
+    db.ETH_data.find({
+        date: {
+            $gte: req.params.start_date,
+            $lt: req.params.end_date
+        }
+    }).exec(function(err, data) {
+        if (err) {
+            console.log(err)
+            next(err);
+        } else if (data) {
+            var result = data.filter(function(el, index) {
+                return index % 2 === 0;
+            });
+            res.json({ error: 0, message: "data found", data: result });
+        } else {
+            res.json({ error: 1, message: "data not found", data: result });
+        }
+    })
+})
+
+router.get('/get_xrp/:start_date/:end_date', function(req, res, next) {
+    db.XRP_data.find({
+        date: {
+            $gte: req.params.start_date,
+            $lt: req.params.end_date
+        }
+    }).exec(function(err, data) {
+        if (err) {
+            console.log(err)
+            next(err);
+        } else if (data) {
+            var result = data.filter(function(el, index) {
+                return index % 2 === 0;
+            });
+            res.json({ error: 0, message: "data found", data: result });
+        } else {
+            res.json({ error: 1, message: "data not found", data: result });
+        }
+    })
+})
+
+router.get('/get_ltc/:start_date/:end_date', function(req, res, next) {
+    db.LTC_data.find({
+        date: {
+            $gte: req.params.start_date,
+            $lt: req.params.end_date
+        }
+    }).exec(function(err, data) {
+        if (err) {
+            console.log(err)
+            next(err);
+        } else if (data) {
+            var result = data.filter(function(el, index) {
+                return index % 2 === 0;
+            });
+            res.json({ error: 0, message: "data found", data: result });
+        } else {
+            res.json({ error: 1, message: "data not found", data: result });
+        }
+    })
+})
+
+router.get('/get_bch/:start_date/:end_date', function(req, res, next) {
+    db.BCH_data.find({
+        date: {
+            $gte: req.params.start_date,
+            $lt: req.params.end_date
+        }
+    }).exec(function(err, data) {
+        if (err) {
+            console.log(err)
+            next(err);
+        } else if (data) {
+            var result = data.filter(function(el, index) {
+                return index % 2 === 0;
+            });
+            res.json({ error: 0, message: "data found", data: result });
+        } else {
+            res.json({ error: 1, message: "data not found", data: result });
+        }
+    })
+})
+
+router.get('/seperate_collections', function(req, res, next) {
```

Comment:
can you explain purpose of this route?

Reply:
sir it was for storing old data into new collections. now there is no use of it. i will remove it